asdl/runtime.cc

```diff
@@ -4,19 +4,18 @@
 
 #ifdef LEAKY_BINDINGS
 #include "mycpp/mylib_old.h"
-using gc_heap::Alloc;
-using gc_heap::kZeroMask;
+
```

Comment:
Oh do you know this file is separately built?

It's `build/translate.sh asdl-runtime`

the shell scripts might need to change, I can help with that

Reply:
Looks like it still builds